app/views/users/_form.html.erb

```diff
@@ -25,7 +25,7 @@
 <!-- *** migration exercise no 6 ***  -->
   <div class="field">
     <%= f.label :email, 'Email:' %>
-    <%= f.email_field :email %>
+    <%= f.text_field :email %>
```

Comment:
this is not related to migrations' exercise.
I have changed it for checking html validations.